students/13th/hamjunho/js/main.js

```diff
@@ -0,0 +1,289 @@
+      nickname: 'weboot',
+      message: '>wecode | 위코드'
+    },
+    {
+      icon: 'https://www.enewstoday.co.kr/news/photo/201906/1313207_382877_2618.jpg',
+      nickname: 'nakedbibi',
+      message: '비비'
+    },
+    {
+      icon: 'https://i.ytimg.com/vi/kIxAcVIhWPE/maxresdefault.jpg',
+      nickname: 'Mr.Song',
+      message: '송강호'
+    },
+    {
+      icon: 'https://img.hankyung.com/photo/201701/03.13255037.1.jpg',
+      nickname: 'jichangwook',
+      message: '지창욱'
+    },
+    {
+      icon: 'https://scontent-gmp1-1.xx.fbcdn.net/v/t1.0-9/29196762_1802345479823070_4967943661961441257_n.jpg?_nc_cat=108&_nc_sid=110474&_nc_ohc=AGTKfJe4HcwAX_dAu_K&_nc_ht=scontent-gmp1-1.xx&oh=30d3886df6b902fb64dc80a269ce463b&oe=5F910193',
+      nickname: 'lilyiu_',
+      message:`아이유(IU) | 이지은(Lee Jieun) 🧡`
+    },
+    {
+      icon: 'https://scontent-gmp1-1.cdninstagram.com/v/t51.2885-19/s320x320/50959409_809292579425612_573364557624377344_n.jpg?_nc_ht=scontent-gmp1-1.cdninstagram.com&_nc_ohc=B9rbwtW2wU8AX-RyzAc&oh=a001eb7f445dd801f901f276a9fdde5e&oe=5F91AAAC',
+      nickname: 'page0blue',
+      message: '귀찮네'
+    },
+    {
+      icon: 'https://ww.namu.la/s/aeca2e14dbb78281beabffe6d5a8b1a84233da4aa2d7b857f11baa8530908b4faf2dae5ce55389c73821b2235cfa3c4d9744bb30d3edc6efda4a446164481c7883a1601fe728564b751b30bb4e7b5fec0c9b9560abda620ddf868d89baf37357aee9213d8393be29d72c5d821151ba8b',
+      nickname: 'EasyGold',
+      message: '이지금'
+    },
+    {
+      icon: 'https://ww.namu.la/s/4e68d2da7f9dc6ec1251daa195275de1a481dbef92a1f00a627dc5f7bdc1aeb8dcfb3dee162cb279e0b28e785e141a99af2559dad5cebfcaa63bad4013cf6edc21d55d1cad4d4fac317161f8f1801c916355ce3711c1a9e4121d7dfc49034ecae954781309df50a86f607b41316f8e26',
+      nickname: 'moonLight',
+      message: '문별'
+    },
+    {
+      icon: 'https://w.namu.la/s/b99033a5fa0125d8bed88db1287d8d6e44ecbaa56f51941c0adc6a9dc5179e6651aeee3e3c7f95bc50d9eb4ac432b731f8bdf6bf941c87d1133b9586e5af736ba7aa52c31a153184c1850b642ae9a52548d876c9ece5e433187a6abfb954741b34400921306aa3f85b9657054c813377',
+      nickname: 'jiminstart',
+      message: '정정'
+    },
+  ]
+}
+
+const dummyPost = {
+  user: {
+    nickname: 'page0blue',
+    icon: 'https://scontent-gmp1-1.cdninstagram.com/v/t51.2885-19/s320x320/50959409_809292579425612_573364557624377344_n.jpg?_nc_ht=scontent-gmp1-1.cdninstagram.com&_nc_ohc=B9rbwtW2wU8AX-RyzAc&oh=a001eb7f445dd801f901f276a9fdde5e&oe=5F91AAAC'
+  },
+  postId: 0,
+  images: {
+    src: 'https://file.mk.co.kr/meet/neds/2020/04/image_readtop_2020_435990_15879677954178860.png'
+  },
+  contents: '날씨가 너무 좋아',
+  comments: [
+    {
+      nickname: 'kwe1s',
+      contents: '첫번째 댓글',
+      liked: true,
+    },
+    {
+      nickname: 'kwe2s',
+      contents: '두번째 댓글',
+      liked: false,
+    }
+  ],
+  liked: 100,
+}
+/// dummy 부분 ----
+
+const _map = (f, iter) => { let res = []; for (const a of iter) { res.push(f(a)); } return res; }
```

Comment:
```suggestion
const _map = (f, iter) => {
  let res = [];
  for (const a of iter) {
    res.push(f(a));
  }
  return res;
};
```

- 함수 활용 Good! Array.map 메소드가 존재하는데 배열이 아닌 데이터 타입을 위해 따로 정의하신건가요?
- 가독성을 위해 줄바꿈을 해주시면 좋습니다.
- 저희는 함수 정의할 때 underbar를 사용하지 않습니다. 나중에 팀 프로젝트를 위해 지금부터 camelCase로 함수명 작성해주세요 :) 
- 마찬가지로 `a` , `iter` 와 같이 줄임말도 팀 프로젝트 진행 시에 다른 개발자들에게는 의미가 명확하지 않을 수 있습니다. 조금 길어도 괜찮으니 좀 더 명확히 드러나는 의미를 써주시면 좋습니다.